documentation/ParserCombinators.md

```diff
@@ -160,5 +160,5 @@ is built.  All of the following parsers consume characters acquired from
 
 ### Debugging Parser
 Last, a string literal `"..."_debug` denotes a parser that emits the string to
-`std::cerr` and succeeds.  It is useful for tracing while debugging a parser but should
+`llv::errs` and succeeds.  It is useful for tracing while debugging a parser but should
```

Comment:
Is this really llv?  Sorry I didn't look it up.

Reply:
Sorry @sscalpone. It looks like I cannot differentiate 2 l's from one l anymore. :(
I've looked too much for this.

Reply:
Done